src/model.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+from market_data import MarketData
+from enums import *
+from volgrid import VolGrid
+import numpy as np
+from abc import ABC, abstractmethod
+
+class MarketModel(ABC):
+    def __init__(self, und: Stock):
+        self._und = und
+        self._interest_rate = MarketData.get_interest_rate()
```

Comment:
I missed this one from MarketData. I've just added in PR #12, but named it as _get_risk_free_rate()_ as it's more meaningful. Please rename each occurence.